src/ch00-10-events.md

```diff
@@ -0,0 +1,9 @@
+# Events
+
+An event is defined as function with the #[event] attribute. The parameters of the function correspond to data that will be emitted. An event is to be logged for easy and fast access to querying the data at a later time.
+
+Here's a simple example of a contract using events to log a counter that increments:
```

Comment:
>  of a contract using events to log a counter that increments:

be more specific: of a contract using that emits an event each time a counter is incremented